src/clients/c++/request_common.cc

```diff
@@ -905,6 +905,64 @@ InferContextImpl::GetOutput(
       "unknown output '" + name + "' for '" + model_name_ + "'");
 }
 
+int64_t
+InferContextImpl::ByteSize(const DimsList& dims, const DataType& dtype) const
```

Comment:
You can include src/core/model_config.h and just use the utility routine from there.

Reply:
Sure i'll do that